<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-NEO5FTIU" failures="1" tests="1" name="Test" time="8.19" errors="0" timestamp="2022-01-02T16:46:10 EST">
  <testcase classname="Demo.demo1" name="pub" time="0.46">
    <failure type="java.net.MalformedURLException">
      <![CDATA[java.net.MalformedURLException
at java.base/java.net.URL.<init>(URL.java:679)
at java.base/java.net.URL.<init>(URL.java:541)
at java.base/java.net.URL.<init>(URL.java:488)
at Demo.demo1.pub(demo1.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
Caused by: java.lang.NullPointerException
at java.base/java.net.URL.<init>(URL.java:585)
... 31 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- pub -->
</testsuite> <!-- Test -->
